Bank-admin-API-design.md

```diff
@@ -0,0 +1,55 @@
+# Bank admin API
+
+## /bank_admin/transactions/ - GET
+
+Returns a list of all recent transactions
+
+### Filters
+
+**status**
+
+Filters by the status of the transactions, accepts multiple values which will be 
+OR'd.
+
+Possible values:
+- *available*: returns list of transactions that have not been locked by anyone
+- *locked*: returns list of locked transactions
+- *credited*: returns list of transactions credited
+- *refunded* (*not implemented yet*): returns list of transactions refunded
+- *refund_pending* (*not implemented yet*): returns list of transactions in pending refund
+
+For ADI file generation, this will likely be:
+
+?status=credited&status=refunded
```

Comment:
I just don't like trying to render data structures in text fields. In this case it is unlikely that any problems will arise as the values of status are limited, so if the & bothers you I will change it.